firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -80,7 +80,31 @@ const TCMD_TelecommandDefinition_t TCMD_telecommand_definitions[] = {
         .number_of_args = 0,
         .readiness_level = TCMD_READINESS_LEVEL_FOR_OPERATION,
     },
-
+    {
+        .tcmd_name = "set_int_config_var",
+        .tcmd_func = TCMDEXEC_set_int_config_variable,
+        .number_of_args = 2,
+    },
+    {
+        .tcmd_name = "set_str_config_var",
+        .tcmd_func = TCMDEXEC_set_str_config_variable,
+        .number_of_args = 2,
+    },
+    {
+        .tcmd_name = "get_int_config_var",
+        .tcmd_func = TCMDEXEC_get_integer_configuration_variable,
+        .number_of_args = 1,
+    },
+    {
+        .tcmd_name = "get_string_config_var",
```

Comment:
All the rest are `int` and `str`, but this one is `string`. Maybe rename to `str` for consistency.